scripts/make_port_status.py

```diff
@@ -113,12 +113,21 @@ def get_mathlib4_module_commit_info(contents):
 prs = {}
 fetch_args = ['git', 'fetch', 'origin']
 nums = []
+
+sync_prs = dict()
+labels = dict()
+
 mathlib4repo = github.Github(github_token).get_repo("leanprover-community/mathlib4")
 for pr in mathlib4repo.get_pulls(state='open'):
     if pr.created_at < datetime.datetime(2022, 12, 1, 0, 0, 0):
         continue
     if 'no-source-header' in (l.name for l in pr.labels):
         continue
+    if 'mathlib3-pair' in (l.name for l in pr.labels):
+        for file in (f.filename for f in pr.get_files()):
+            sync_prs[file] = sync_prs.get(file, set()).union([pr.number])
+    if 'mathlib-port' in (l.name for l in pr.labels):
+        labels[pr.number] = [[l.name, l.color] for l in pr.labels]
```

Comment:
let's make the format a bit more verbose so that it's obvious
```suggestion
        labels[pr.number] = [dict(name=l.name, color=l.color) for l in pr.labels]
```
size of this file doesn't really matter.